blog/templates/blog/viewpost.html

```diff
@@ -31,15 +31,44 @@ <h2>{{ post.title }}</h2>
 	</div>
 </div>
 
+
+{% if user and user.is_authenticated and user.author%}
 <div class="post-comments">
     <h1>Comments</h1>
-	<form id = 'comment' action="" method="post" >
-	<textarea id="com">
-	</textarea>
+	<form id = 'comment' action="{{comment}} " method="post" >
```

Comment:
i'm not sure how this works, but you should use django reverse urls:
`action="{ url 'addComment' post.pk}"`